docs/qix/objects.md

```diff
@@ -4,12 +4,19 @@ An object, or a generic object, binds to data using hypercubes, listobjects and
 
 A generic object has a hierarchic structure, a data structure and properties, and after it has been calculated, it also has a layout.
 
-
-## Properties
+## Properties and layout
 
 Generic objects has properties in a structure that you define yourself. There are two kinds of properties:
-- User defined: persisted by Qlik Sense and included as-is in the layout.
-- QIX defined: validated by the Qlik engine and replaced in the output, that is the layout, by the calculated counterparts.
+- QIX defined: validated by the Qlik engine and replaced in the output, that is the layout, by the calculated counterparts. These properties are prefixed with `q`, such as `qId` and `qHyperCubeDef`.
+- User defined: Properties without the `q`-prefix are _dynamic properties_. They are persisted by Qlik Sense and included as-is in the layout.
```

Comment:
Would be good to add a link to the help section regarding the properties concept i.e https://help.qlik.com/en-US/sense-developer/3.2/Subsystems/Platform/Content/Concepts/Properties.htm